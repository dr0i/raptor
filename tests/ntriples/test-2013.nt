#
# N-Triples 2013 Test Cases
# Dave Beckett - http://purl.org/net/dajobe/
# 
# comment lines
         	
<http://example.org/resource1> <http://example.org/property1> <http://example.org/resource2> .  # comment at end of line
<http://example.org/resource1> <http://example.org/property2> "Cette section définit un format facile à analyser la ligne à base de sous-ensemble de tortue nommée N-Triples" .
