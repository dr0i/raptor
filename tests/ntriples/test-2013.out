<http://example.org/resource1> <http://example.org/property1> <http://example.org/resource2> .
<http://example.org/resource1> <http://example.org/property2> "Cette section d\u00E9finit un format facile \u00E0 analyser la ligne \u00E0 base de sous-ensemble de tortue nomm\u00E9e N-Triples" .
